src/main.cpp

```diff
@@ -1686,6 +1688,43 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
     return res;
 }
 
+bool ReadTransaction(CTransaction& tx, const CDiskTxPos &pos, uint256 &hashBlock) {
```

Comment:
Lack of C++ unit test (this may be deficit of merge, I don't know - but it looks suitable for unit testing).

Same applies to FindTransactionByDestination() below.